src/Servers/Connections.Abstractions/src/ConnectionDelegate.cs

```diff
@@ -5,5 +5,10 @@
 
 namespace Microsoft.AspNetCore.Connections
 {
+    /// <summary>
+    /// A function that can process an connection.
```

Comment:
```suggestion
    /// A function that can process a connection.
```

Somebody copy-pasta'd from https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.requestdelegate?view=aspnetcore-2.2 ;)